doc/user_guide/dynamo0p3.rst

```diff
@@ -1870,22 +1881,47 @@ The basic concept of a PSyclone Built-in is described in the
 :ref:`built-ins` section.  In the Dynamo0.3 API, calls to
 Built-ins generally follow a convention that the field/scalar written
 to comes first in the argument list. Dynamo0.3 Built-ins must conform to the
-following four rules:
+following rules:
 
 1) Built-in kernels must have one and only one modified (i.e. written
    to) argument.
 
-2) There must be at least one field in the argument list. This is so
+2) The iteration space of a Built-in kernel must be over DoFs
```

Comment:
No need to say "Built-in kernel" here as this list is only for Built-ins. (Same in point 1) although that's not your fault.)

Reply:
Changed.